tools/viewsvg/svgview.cpp

```diff
@@ -44,6 +47,11 @@ QString SvgViewWorker::loadFile(const QString &path)
     return QString();
 }
 
+void SvgViewWorker::setScreenDPI(const QScreen *screen)
+{
+    m_dpiRatio = screen->devicePixelRatio();
```

Comment:
I tried to catch a nullpointer here, but it doesn’t help with the crashes. Probably still a good idea…